### PR TITLE
AVX-39042: deprecate keep_alive_via_lan_interface_enabled option

### PR DESCRIPTION
**Pull request by shanxu2017**
Disable keep_alive_via_lan_interface_enabled option in Firenet resource.

### CODE REVIEW

**Review by inc1t3Ful — CHANGES_REQUESTED** [review]
LGTM-

Please also update the guides/feature-changelist-v3.md for the R3.2.0 section for noting the deprecation. 

You can use the R3.1.4 section as a reference, as the deprecation was mentioned before. Now this release, we can say it is fully deprecated/removed. 

nit: as an aside, please also remove the trailing duplicate "**" characters in the 3.1.4 and 3.1.5 section headers

**Comment by shanxu2017** [comment]
> LGTM-
> 
> Please also update the guides/feature-changelist-v3.md for the R3.2.0 section for noting the deprecation.
> 
> You can use the R3.1.4 section as a reference, as the deprecation was mentioned before. Now this release, we can say it is fully deprecated/removed.
> 
> nit: as an aside, please also remove the trailing duplicate "**" characters in the 3.1.4 and 3.1.5 section headers

@inc1t3Ful can you take a look again?

File: docs/guides/feature-changelist-v3.md [hunk 1]
```diff
@@ -92,8 +92,4 @@ The following logging resources are removed:
 | Diff | Resource | Attribute | Action Required? |
 |:----:|:--------:|:---------:|:----------------:|
 |(deprecated) | edge_equinix, edge_equinix_ha, edge_csp, edge_csp_ha, edge_neo, edge_neo_ha, edge_platform, edge_platform_ha, edge_zededa, edge_zededa_ha | bandwidth | **Yes**; This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release |
-
-
-
-
-
+|(deprecated) | firenet | keep_alive_via_lan_interface_enabled | **Yes**; This configuration value no longer has any effect. It will be removed from the Aviatrix provider in the 3.2.0 release |
```
Comment (inc1t3Ful) [inline docs/guides/feature-changelist-v3.md:95]: Can this be under its own table/header?

If this is for 7.1, it should be R3.1.6
If this is for 7.2, it should be R3.2.0

**Review by inc1t3Ful — CHANGES_REQUESTED** [review]

File: docs/guides/feature-changelist-v3.md (hunk 1 above)
Comment (shanxu2017) [reply docs/guides/feature-changelist-v3.md:95]: This is under ## R3.2.0 (UserConnect-7.2)**

Comment (inc1t3Ful) [reply docs/guides/feature-changelist-v3.md:95]: Oh my mistake- the way the diff shows was a little confusing. Ok! LGTM

**Review by inc1t3Ful — APPROVED** [review]

**Pull request merged**